p2panda-js/package.json

```diff
@@ -8,10 +8,12 @@
     "build": "run-s clear wasm webpack",
     "clear": "rimraf ./lib ./wasm-web ./wasm-node",
     "lint": "eslint --ext .ts .",
-    "test": "run-s clear test:prepare test:mocha",
+    "mock-node-rpc": "open-rpc-mock-server --port 2020 > ./mock-server.log",
```

Comment:
Storing the log into a file is cool but maybe that option should be left to the user, in case one prefers to see the logs in a terminal or anywhere else. If I need to see it in a file I can still manually add the redirect when executing the npm script?